docker/base-images/base-builder.Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM golang:1.18.0-alpine3.15 AS base-builder
+FROM golang:1.18-alpine3.15 AS base-builder
```

Comment:
1.18.2?

Reply:
Considered this.
Pros:
- exact version is pinned, and you know for sure what do you use
Cons:
- with every patch release we will need to update this files instead of just rebuild.

And then alpine itself should be referred as `1.15.4` but it is already referred as `1.15` by [golang](https://github.com/docker-library/golang/blob/136318870f2f0837098f0ff9d230336546721750/1.18/alpine3.14/Dockerfile#L7), so even if we specify all patch versions on our side, we still will be in some non-determined state. And the only way to figure out if some specific patch is included or not is to co compare publish dates on docker hub.

Reply:
generally you only pin a patch version if the newest version has a problem.